.github/ISSUE_TEMPLATE/NEW_COMPONENT.yml

```
@@ -0,0 +1,53 @@
+name: New Component
+description: Provide details to help define a new component for the system
+labels: [feature request, pending triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **⚠️ This is a public repository. Please be aware of sensitive information and prefer referencing internal links.**
+
+        Please leverage this template to provide as much details about the visual and functional requirements for this new component. Doing so will help use communicate and collborate on a solution effectively.
+
+  - type: text
+    attributes:
+      label: Design File
+      description: Provide a Figma link as a reference
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Description
+      description: |
+        Provide as much details as possible about the visual and functional requirements.
+      placeholder: |
+        <!--
+          Consider UI/UX like affordance, user interaction, and different states
+        -->
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Research
+      description: |
+        How similar is this new component to any existing component?
```

This sounds like you're asking for similarities between other components in Maker.

And specifically, I don't think we care about "how similar" it is to other 3rd party components, but we just want references.

```suggestion
        Are there similar components in 3rd party component libraries? 
```

---

I was attempting to do both. For example, there's a new component getting queued up for "pin codes" that are very similar to visual and functional requirements of an input or password component. Referencing our existing input component would likely be a good start.  However, some of the functional requirements are still new and 3rd party references are relevant too.

---

Changed to be a bit more specific.